packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/AssetViewOnly/AssetViewOnly.tsx

```diff
@@ -0,0 +1,451 @@
+                        <Description>
+                          <Flex flexDirection='column'>
+                            <Text size='16px' color='grey900' weight={600}>
+                              <FormattedMessage
+                                id='copy.description'
+                                defaultMessage='Description'
+                              />
+                            </Text>
+                            <Text
+                              size='16px'
+                              color='grey900'
+                              weight={500}
+                              style={{ wordBreak: 'break-word' }}
+                            >
+                              {moreAssetToggle && asset.description.length > 82 ? (
+                                <ReactMarkdown linkTarget='_blank'>
+                                  {`${asset?.description.substring(0, 82)}...`}
+                                </ReactMarkdown>
+                              ) : (
+                                <ReactMarkdown linkTarget='_blank'>
+                                  {asset.description}
+                                </ReactMarkdown>
+                              )}
+                            </Text>
+                            {asset?.description?.length > 82 && (
+                              <Text
+                                style={{ cursor: 'pointer' }}
+                                onClick={() => {
+                                  if (asset.description?.length > 82)
+                                    setMoreAssetToggle(!moreAssetToggle)
+                                }}
+                                size='16px'
+                                color='blue600'
+                                weight={600}
+                              >
+                                {moreAssetToggle ? (
+                                  <FormattedMessage id='copy.more' defaultMessage='See More' />
+                                ) : (
+                                  <FormattedMessage id='copy.less' defaultMessage='Less' />
+                                )}
+                              </Text>
+                            )}
+                          </Flex>
+                        </Description>
+                      ) : null}
+                      <MarginButton
+                        onClick={() =>
+                          nftsActions.nftOrderFlowOpen({
+                            asset_contract_address: contract,
+                            step: NftOrderStepEnum.TRANSFER,
+                            token_id: id
+                          })
+                        }
+                        fullwidth
+                        data-e2e='transfer-nft'
+                        nature='primary'
+                      >
+                        Transfer NFT
+                      </MarginButton>
+                      {asset?.traits.length ? (
```

Comment:
I think if you do `{asset && asset.traits && asset.traits.length ? ...` here you don't have to add the `?` inside of this anymore. Typescript will know that it's defined inside of this block.